Added support for svmon on pakiti.

Agupane opened the pull request:
SVMON collects the information on software versions of EUDAT services and their components installed in EUDAT CDI. The framework allows the monitoring of the following information:
-Operating system flavor and version of the endpoint.
-EUDAT service component software versions.
-Any service software version installed on the endpoint.
-History of changes performed for the given service.

Agupane committed:
Added support for svmon on pakiti.

SVMON collects the information on software versions of EUDAT services and their components installed in EUDAT CDI. The framework allows the monitoring of the following information:
-Operating system flavor and version of the endpoint.
-EUDAT service component software versions.
-Any service software version installed on the endpoint.
-History of changes performed for the given service.

kouril commented on bin/pakiti-client:
```diff
@@ -238,6 +238,25 @@ sub find_packages ($) {
+    if ($Option{"svmonreport"}) {
+        $cmd = $Option{"svmon"};
+        $data->{packager} = "svmon";
+        @output = qx($cmd -p) or die("$Script: No svmon found on path $cmd\n");
```
"Failed to execute the command $cmd" is more appropriate message here.

kouril commented on bin/pakiti-client:
```diff
@@ -238,6 +238,25 @@ sub find_packages ($) {
+	
+    #The user wants to send svmon data
+    if ($Option{"svmonreport"}) {
+        $cmd = $Option{"svmon"};
```
You probably want to make the user's life easier and check that the $cmd is non-empty here (and possibly trigger an error). Otherwise the code fails later with a potentially cryptic message (when -s is specified and svmon isn't in the PATH).

kouril commented on bin/pakiti-client:
```diff
@@ -452,6 +471,8 @@ sub init () {
         "rndsleep" => "|r=i",
         "site"     => "=s",
+        "svmonreport" => "|s",
+        "svmon"    => "=s",
```
This specification is superfluous, it's added automatically later (see line 479), please remove it.

Agupane committed:
Added support for svmon on pakiti.

SVMON collects the information on software versions of EUDAT services and their components installed in EUDAT CDI. The framework allows the monitoring of the following information:
-Operating system flavor and version of the endpoint.
-EUDAT service component software versions.
-Any service software version installed on the endpoint.
-History of changes performed for the given service.
